modules/custom/az_marketing_cloud/config/install/views.view.az_marketing_cloud.yml

```diff
@@ -1032,6 +1032,63 @@ display:
           hide_empty: false
           empty_zero: false
           hide_alter_empty: false
+        delta:
+          id: delta
+          table: node__field_az_event_date
+          field: delta
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: numeric
+          label: ''
+          exclude: true
+          alter:
+            alter_text: false
+            text: ''
+            make_link: false
+            path: ''
+            absolute: false
+            external: false
+            replace_spaces: false
+            path_case: none
+            trim_whitespace: false
+            alt: ''
+            rel: ''
+            link_class: ''
+            prefix: ''
+            suffix: ''
+            target: ''
+            nl2br: false
+            max_length: 0
+            word_boundary: true
+            ellipsis: true
+            more_link: false
+            more_link_text: ''
+            more_link_path: ''
+            strip_tags: false
+            trim: false
+            preserve_tags: ''
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: ''
+          element_label_class: ''
+          element_label_colon: false
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          set_precision: false
+          precision: 0
+          decimal: .
+          separator: ','
+          format_plural: false
+          format_plural_string: !!binary MQNAY291bnQ=
```

Comment:
More background on this issue with `format_plural_string`: https://www.drupal.org/project/drupal/issues/2829919.

I don't believe this issue needs to block this PR, especially since `format_plural` is false in both cases in this file (as @danahertzberg noted).

Reply:
I agree - it's a bit of a symfony encoding oddity but doesn't seem to have any actual effect. Thanks @bberndt-uaz !